string filename
LineSegmentArray Path
---
LineSegmentArray Path
ImpingementArray Impingement
